modules/integration/tests-ui/src/test/resources/appconfig.properties

```diff
@@ -3,9 +3,17 @@ appVersion = 1.0.0
 appURL = http://localhost
 appTransport = http
 appName = testApp
+appDisplayName = testApp
 tier = Bronze
 tags = youtubeApi
 role = internal/reviewer
 path = /sample
 appSample = ./samples/sample.war
-policyFile = ./samples/policy.xml
\ No newline at end of file
+policyFile = ./samples/policy.xml
+policyGroupName = test
+policyGroupDesc =null
```

Comment:
This code segment has been refactored and this code line is not in here any more.